idea_v16.properties

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright 2000-2013 JetBrains s.r.o.
+# Copyright 2014-2016 AS3Boyan
+# Copyright 2014-2014 Elias Ku
```

Comment:
No, I've started maintaining plugin with @as3boyan in 2014, then get delayed, and returned by the end of 2015, so 2014, 2016 will be correct :)